common4j/src/main/com/microsoft/identity/common/java/providers/microsoft/azureactivedirectory/AzureActiveDirectoryAccessToken.java

```diff
@@ -41,22 +41,22 @@
     public AzureActiveDirectoryAccessToken(
             @NonNull final AzureActiveDirectoryTokenResponse response) {
         super(response);
-        mExpiresOn = response.getExpiresOn();
-        mExtendedExpiresOn = response.getExtExpiresOn();
+        mExpiresOn = response.getExpiresOn() == null ? null : new Date(response.getExpiresOn().getTime());
```

Comment:
the response is already doing one defensive copy.  Do we need to copy it again, since we're not storing it in this class?